raddb/mods-available/cache

```diff
@@ -174,6 +174,11 @@ cache {
 	#
 	add_stats = no
 
+	#
+	#  max_entries:: Maximum number of retries.
+	#
+#	max_entries = 0
```

Comment:
That's not the maximum number of retries, that's the maximum number of entries allowed in the rbtree driver